### PR TITLE
Small updates, mostly to build instructions

### PR DESCRIPTION
**Pull request by sungeunchoi**

### CODE REVIEW

**Comment by sungeunchoi** [comment]
@tonyzinger

File: scripts/miniapps/HPGMG.md [hunk 1]
```diff
@@ -33,7 +32,8 @@ and set the recommended environment variables, e.g. for MPICH, etc.
    % make -j3 -C build
```
Comment (tonyzinger) [inline scripts/miniapps/HPGMG.md:32]: All of the other READMEs, you added to the "Build" line, "(in the directory xxx)".
You should add it here to make them consistent.

**Review by tonyzinger — APPROVED** [review]
One minor comment.

File: scripts/miniapps/HPGMG.md (hunk 1 above)
Comment (sungeunchoi) [reply scripts/miniapps/HPGMG.md:32]: Fixed.

**Pull request merged**